robottelo/constants.py

```diff
@@ -245,6 +246,8 @@
     'rhaht': 'Red Hat Enterprise Linux Atomic Host (Trees)',
     'rhdt7': ('Red Hat Developer Tools RPMs for Red Hat Enterprise Linux 7'
               ' Server'),
+    'rhscl7': ('Red Hat Software Collections RPMs for Red Hat Enterprise'
+              ' Linux 7 Server'),
```

Comment:
add a single space in front of the first ' to align these for flake.

Reply:
Added some spaces here and there ;-) Should be better now. I have not tested it successfully yet still.